docs/source/user/aerodyn/theory_ua.rst

```diff
@@ -197,8 +197,12 @@ Two variants are implemented in the Unsteady Aerodynamic module. These two (comp
 Beddoes-Leishman 4-states model (UAMod=4)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The 4-states (incompressible) dynamic stall model from Hansen-Gaunaa-Madsen (HGM) is described in :cite:`ad-Hansen:2004` and enabled using ``UAMod=4``.  The model uses :math:`C_l` as main physical quantity. 
-Linearization of the model will be available in the future.
+The 4-states (incompressible) dynamic stall model as implemented in OpenFAST is described in :cite:`ad-Branlard:2022`. 
+The original formulation from Hansen-Gaunaa-Madsen (HGM) is described in :cite:`ad-Hansen:2004`.
+This model is enabled using ``UAMod=4``.  The model uses :math:`C_l` as main physical quantity. 
```

Comment:
This seems contradictory. Presumably UAMod = 4 provides the update to the HGM model as documented in our TORQUE 2022 paper.